go.mod

```diff
@@ -0,0 +1,35 @@
+module github.com/mailhog/MailHog
+
+go 1.6
```

Comment:
is there a reason for 1.6 - seems very old?

Reply:
Does "old" necessarily mean "bad"?  If it's supported it's supported, right?

Reply:
old often means "has security issues that are fixed in more recent versions"
(alas, the same could be said of *this* project, so I should be careful, here! 😉 )

Reply:
For some reason I assumed the Go 1 compat promise meant security fixes would be backported to 1.x, but apparently I'm a dumbass, and only the previous two releases receive security updates.

Disregard.

Reply:
I was thinking of keeping it the lowest possible version for compatibility, but you are right about security fixes, so I'' up it to 1.17